shared/chat/conversation/input-area/normal/platform-input.desktop.js

```diff
@@ -279,6 +291,36 @@ class PlatformInput extends Component<PlatformInputProps, State> {
                   type="iconfont-boom"
                 />
               )}
+            {flags.explodingMessagesEnabled &&
+              this.state.explodingPopupOpen && (
+                <SetExplodingMessagePopup
+                  attachTo={this.props.attachmentRef}
+                  isNew={true}
```

Comment:
@buoyad, how should this be determined?

Reply:
Not yet implemented, I haven't gotten to it yet in the integration